spec/harvesting/harvest_data_setup.rb

```diff
@@ -190,4 +190,29 @@
     result
   end
 
+  # let!(:toto_invoice) do
+  #   VCR.use_cassette('harvest_data_setup/invoice', :record => :once, :allow_playback_repeats => true) do
+  #     invoice = Harvesting::Models::Invoice.new(
+  #       {
+  #         "client" => {
+  #           "id" => client_pepe.id.to_s
+  #         },
+  #         "creator" => {
+  #           "id" => client_toto.id.to_s
+  #         },
+  #         "line_items" => [
+  #           {
+  #             "project" => {
+  #               "id" => project_castle_building.id.to_s
+  #             }
+  #           }
+  #         ]
+  #       },
+  #       client: harvest_client
+  #     )
+  #     invoice.save
+  #     invoice
+  #   end
+  # end
+
```

Comment:
@etagwerker  Do dev's have the permission to create new cassette?

Reply:
@chimedpalden Yes, you will have to use your own Harvest account to create the cassettes. Cassettes are accepted and encouraged for all tests.